dvc/path_info.py

```diff
@@ -57,6 +57,16 @@ def __repr__(self):
     def __fspath__(self):
         return pathlib.PurePath.__str__(self)
 
+    def __eq__(self, other):
```

Comment:
is it needed for this PR? (or just a warning fix?)

Reply:
Sorry, again pressing "Resolve Conversation" without pressing "Comment". Yes, it's needed. I got some cases where the code failed because the relative path was not correctly compared.